server/integration_tests/test_data/usa_map_types/query_2/chart_config.json

```diff
@@ -39,9 +39,24 @@
                 "tiles": [
                   {
                     "statVarKey": [
-                      "Median_Income_Household_HouseholderRaceWhiteAlone"
+                      "Median_Income_Household_HouseholderRaceAsianAlone"
                     ],
-                    "title": "Median Income of White Households in Census Tracts of Placer County (${date})",
+                    "title": "Median Income of Asian Households in Census Tracts of Placer County (${date})",
```

Comment:
This and the next diff comes about because of the tweaks to the tracts maps in this PR.  We consume away "tract" from the query and match more SVs I think...